library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/UnsupportedViewMapper.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.mapper
+
+import android.os.Build
+import android.view.View
+import android.widget.Toolbar
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.model.MobileSegment
+import androidx.appcompat.widget.Toolbar as AppCompatToolbar
+
+internal class UnsupportedViewMapper :
+    BaseWireframeMapper<View, MobileSegment.Wireframe.TextWireframe>() {
+
+    override fun map(view: View, mappingContext: MappingContext):
+        List<MobileSegment.Wireframe.TextWireframe> {
+        val viewGlobalBounds = resolveViewGlobalBounds(
+            view,
+            mappingContext.systemInformation.screenDensity
+        )
+
+        return listOf(
+            MobileSegment.Wireframe.TextWireframe(
+                id = resolveViewId(view),
+                x = viewGlobalBounds.x,
+                y = viewGlobalBounds.y,
+                width = viewGlobalBounds.width,
+                height = viewGlobalBounds.height,
+                shapeStyle = resolveShapeStyle(view),
+                border = resolveBorder(),
+                text = resolveViewTitle(view),
+                textStyle = resolveTextStyle(),
+                textPosition = resolveTextPosition()
+            )
+        )
+    }
+
+    // region Internal
+
+    private fun resolveViewTitle(view: View): String {
+        return if (isToolbar(view)) {
+            return TOOLBAR_LABEL
+        } else {
+            DEFAULT_LABEL
```

Comment:
just an idea: btw, maybe we can do a trick here: check if class name contains `View` - it it is there, probably it is non-obfuscated class name and we can use it directly, if it is not there - probably it is obfuscated class name (or some other naming convention) and we can use default label.